include/qpid/dispatch/cutthrough_utils.h

```diff
@@ -27,7 +27,7 @@
  * 
  * @param msg - Pointer to a stream
```

Comment:
The @param is not an activation object, not a msg

Reply:
Good catch - fixed!